docs/install_PLC_Devices_Southbound.md

```diff
@@ -20,177 +23,163 @@
-  
-   Energy1:
-   - `central-data` 
-   - `central-metadata`
-  
-   Energy2:
-   - `central-data2`
-   - `central-metadata2`
-  
-    
-  ![Cloud_Connector3](graphics/Cloud_Connector_Route.png)
+After deployment, ensure that the connector status is displayed in "green" and that the data source also shows a green status, like so:
 
-"Local Lake" allows you to connect a local MQTT-Broker like the External Databus 
+![Common Configurator 8](graphics/CommonConfigurator_8.png)
 
-5. Add Cloud Connector Clients
-  - Type: `LOCAL_LAKE`
+To check if data is actually being extracted from the PLC, let's integrate IIH Essentials into Common Configurator. And after creating assets and adding the tags, data is shown:
+
+![Common Configurator 11](graphics/CommonConfigurator_11.png)
+
+Now, repeat the same proccess for Energy 2.
+
+## Configure Data Processing
+
+The data collected from the OPC UA Connector is published to the Databus, and using the Flow Creator, we retrieve this data from the Databus and send it to the central Device (Shopfloor-to-Cloud) through the DataXess application, which will be configured in the Northbound.
+
+Please note that for the Flow Creator to function properly, the Acquisition Devices must be strictly named "Energy1" and "Energy2".
+
+### Flow Creator
+
+Aggregate the raw data from the PLC to:
+- Energy
+- Water
+- Pressured Air
+- Produced Bottles 
+
+The aggregated values are published on newly defined topics to prevent collision with OPC UA Connector related topic names
+
+To do this, follow this steps:
+
+1. Import the JSON-File
   
-  - Publish Topic & enter the Databus credentials of the northbound device:
+    For Energy1: [FlowCreator_Energy1](../src/Device_Energy1/FlowCreator_Energy1.json)
+
+    For Energy2: [FlowCreator_Energy2](../src/Device_Energy2/FlowCreator_Energy2.json)
   
-    Energy1:
-    - Metadata: `ie/m/j/simatic/v1/energy1line1:iefc/dp`
-    - Data: `ie/d/j/simatic/v1/energy1line1:iefc/dp/r/line1/default`
     
-    Energy2:
-    - Metadata: `ie/m/j/simatic/v1/energy2line2:iefc/dp`
-    - Data: `ie/d/j/simatic/v1/energy2line2:iefc/dp/r/line2/default`
+  ![FlowCreator1](graphics/Flow_Creator1.png)
+
+2. Double click on a MQTT-Node  
+3. Add IE Databus Credentials
   
-    
-  ![Cloud_Connector4](graphics/Cloud_Connector_ClientStandard.png)
+  ![FlowCreator2](graphics/Flow_Creator2.png)
   
-
-  ![Cloud_Connector5](graphics/Cloud_Connector_Client1.png)
     
-      
-  ![Cloud_Connector6](graphics/Cloud_Connector_Client2.png)
+  ![FlowCreator3](graphics/Flow_Creator3.png)
 
-6. Mark the data and metadata routes an click "Save Route" 
-      
-        
-  ![Cloud_Connector7](graphics/Cloud_Connector_Route1.png)
-    
-      
-  ![Cloud_Connector8](graphics/Cloud_Connector_Route2.png)
+Before deploying the flows, the following topic has to be changed to `ie/d/j/simatic/v1/opcuac1/dp/r/Energy1/default` for Energy 1 and to `ie/d/j/simatic/v1/opcuac1/dp/r/Energy2/default` for Energy 2:
```

Comment:
Can you update the two flow creator json that this step is not required.

Reply:
Done